### PR TITLE
Default point & line size when styling by value

### PR DESCRIPTION
**Pull request by rjimenezda**
This PR adds default values for points & lines styled by a column, instead of calculating it from the fixed size that the user might've selected.

Related tickets:
https://github.com/CartoDB/support/issues/1089
https://github.com/CartoDB/support/issues/1092

### CODE REVIEW

**Comment by rjimenezda** [comment]
You mean some tests related to the fact that defaultRange is moved around? Not just on the dictionary

That makes sense yeah

**Comment by matallo** [comment]
yeah, something in that fashion, continuing your example like range being 1-5 if it is not provided, and 5-20 in case it is

sorry for being this nitpicking

**Comment by rjimenezda** [comment]
Pushed some specs @matallo 💋

File: lib/assets/core/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-dialog-content.js [hunk 1]
```diff
@@ -105,7 +105,8 @@ module.exports = CoreView.extend({
 
   _updateRangeValue: function () {
     if (this.model.get('fixed') !== null && this.model.get('fixed') !== undefined && this.model.get('attribute')) {
-      var range = this.options.editorAttrs.defaultRange || [this.model.get('fixed'), this.model.get('fixed')];
+      var editorAttrs = this.options.editorAttrs;
+      var range = (editorAttrs && this.options.editorAttrs.defaultRange) || [this.model.get('fixed'), this.model.get('fixed')];
```
Comment (matallo) [inline lib/assets/core/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-dialog-content.js]: `editorAttrs && editorAttrs.defaultRange`

you'll gonna kill me

**Comment by matallo** [comment]
you're the master 👑

File: lib/assets/core/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-dialog-content.js (hunk 1 above)
Comment (rjimenezda) [reply lib/assets/core/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-dialog-content.js]: Oops

Comment (rjimenezda) [reply lib/assets/core/javascripts/cartodb3/components/form-components/editors/fill/input-number/input-number-dialog-content.js]: Wouldn't fail, tho. Just very stupid

**Comment by rjimenezda** [comment]
@matallo I've kept the `defaultRange` only on the dialog-content. Now the input-number-content-view doesn't know anything about it.

On the previous commit, we had a little bug: the dialog would overwrite the range with the defaultRange on each render. Now it only changes it if min === max, the old behaviour. This covers the scenario where we haven't specified a defaultRange for this type of input AND the silly case where the user defines min === max.

I added a spec that will fail if a render causes an unnecessary range change and one that checks that for a range where min === max, a render causes the range to be recalculated. I had to remove some of the previously added ones, which make no sense now.

I think it's simpler now.

**Pull request merged**